account_lock_settings_type.go

```diff
@@ -0,0 +1,13 @@
+package zoom
+
+// AccountLockSettings represents account lock settings
+type AccountLockSettings struct {
+	ScheduleMeeting   interface{} `json:"schedule_meeting,omitempty"`
+	InMeeting         interface{} `json:"in_meeting,omitempty"`
+	EmailNotification interface{} `json:"email_notification,omitempty"`
+	ZoomRooms         interface{} `json:"zoom_rooms,omitempty"`
+	Recording         interface{} `json:"recording,omitempty"`
+	Telephony         interface{} `json:"telephony,omitempty"`
+	TSP               interface{} `json:"tsp,omitempty"`
+	MeetingSecurity   interface{} `json:"meeting_security,omitempty"`
```

Comment:
Can you please add proper types for these? If they not clearly listed in the Zoom documentation, you can contact their dev support and they should be able to tell you or update.